src/py42/clients/archive.py

```diff
@@ -92,8 +92,11 @@ def stream_from_backup(
             private_password=archive_password,
             encryption_key=encryption_key,
         )
+        backup_set_id = self.get_backup_sets(
+            device_guid, archive_accessor.destination_guid
+        )[u"backupSets"][0][u"backupSetId"]
```

Comment:
Need to understand, Why only the first `backupSetId`? is it the latest `backupSetId`?

Reply:
It seems -1 works as backup set ID when you don't need to know it, so I switched to just use -1

Reply:
do we know if this works as expected when we use v3 archives? Will v3 archives require that you specify backup set(s)? @timabrmsn ?

Reply:
I'm not sure, as I've never poked at v3 archive restore. Our internal Code42 clients should have a v3 destination, so we could test using one of our own accounts.

Reply:
Because of Okta, I don't think we can use py42 for our own accounts.

Reply:
> Because of Okta, I don't think we can use py42 for our own accounts.

@unparalleled-js true, but we should be able to login, do a restore, and see in the console if the request behaves any differently for v3

As an aside Okta just ends up creating a v3_user_token, so with some breakpoint trickery or just testing the api without py42 you could still test this with an Okta-based account

Reply:
Ah that makes sense. Thankfully we found a testing environment with V3 archives in it so we don't have to do that